cocoa/src/toga_cocoa/window.py

```diff
@@ -45,10 +46,24 @@ def windowShouldClose_(self, notification) -> bool:
 
     @objc_method
     def windowDidResize_(self, notification) -> None:
-        if self.interface.content:
+        if getattr(self, "interface", None) is not None and self.interface.content:
```

Comment:
I don't think it's getting the events before the interface has been set, rather it is getting them after the references to `self.interface` and `self.impl` are lost i.e., after the window close call to the OS and before the OS actually closes the window.

I am not sure but I think we need to detach the window delegate before closing the window, so that any window delegate events are not transmitted after we call close on the window.